src/components/Page/tests/Page.test.tsx

```diff
@@ -5,7 +5,9 @@ import {Page, DisplayText, Card} from 'components';
 import {noop} from '../../../utilities/other';
 import {LinkAction} from '../../../types';
 import {Header} from '../components';
+// eslint-disable-next-line shopify/strict-component-boundaries
```

Comment:
I'm not 100% sure what the right fix for this is, I think it might be pulling these types up into a higher level, as they're referenced in some other places too. But that's a fix more complex than what I want to do in this PR